fedimint-server/src/config/api.rs

```diff
@@ -521,7 +521,12 @@ impl HasApiContext<ConfigGenApi> for ConfigGenApi {
         request: &ApiRequestErased,
         id: Option<ModuleInstanceId>,
     ) -> (&ConfigGenApi, ApiEndpointContext<'_>) {
-        let dbtx = self.db.begin_transaction().await;
+        let mut db = self.db.clone();
+        let mut dbtx = self.db.begin_transaction().await;
```

Comment:
Should we just get rid of the reference to the dbtx and create a new transaction on every API call?

Reply:
> create a new transaction on every API call

that is already the case

>  get rid of the reference

do you mean remove the `dbtx` argument?

Reply:
> do you mean remove the dbtx argument?

yeah this is what I meant

Reply:
I tried that, ran into some lifetimes issue. It is likely possible. But not sure if it is useful.